Makefile

```diff
@@ -1,4 +1,27 @@
+E2Espec = test/e2e/specs/protractor.conf.js
 WEBAPPSDIR?=/var/www/html/deriva-webapps
 
+.PHONY : install
 install:
 	rsync -avz --exclude='.*' --exclude='Makefile' . $(WEBAPPSDIR)
+
+#This make target for getting the test dependencies only needs to be run once
+.PHONY : testsetup
+install:	
```

Comment:
make sure to change the target here as well

Reply:
@jrchudy I just changed it. Sorry about it